### ISSUE TITLE
enhancing the collection dispatcher to include text-level criteria

### ISSUE THREAD

**Issue by balmas** [Capitains/MyCapytain#185]
It would be nice to be able to use text-level criteria in the collection dispatcher. For example, to be able to create a collection that only includes latin or greek texts (as opposed to their translations)

@PonteIneptique if you agree that this is an okay change to make you could assign it to me.

**Comment by PonteIneptique** [comment]
Yes to it. This is gonna be a pretty massive change though. Good luck.

Le ven. 1 févr. 2019 à 4:49 PM, Bridget Almas <notifications@github.com> a
écrit :

**Comment by balmas** [comment]
thanks for the vote of confidence...
